README.md

```diff
@@ -6,6 +6,9 @@ are the official versions that people should be using.
 See [the governance document](https://jenkins.io/project/governance/) for the background.
 
 # How to file CLA
+
+WARNING: Replace by EasyCLA
```

Comment:
Lets link to the new spot, instead of making it an exercise for the user to figure out.